src/components/PreviewOverlay/PreviewOverlay.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { useContext } from "react";
+import { EpisodesContext as Episodes } from "COMPONENTS/Episodes/";
+import { TrailerContext as Trailer } from "COMPONENTS/Episode/";
+import { s } from "./";
+
+export default function PreviewOverlay(props){
+	
+	// PROPS
+	// ---------------------------
+	const {
+		episodeId,
+		index,
+		episodeNo,
+		title,
+		previewId,
+		trailerSrc,
+		prevEpisode, //bound callback
+		nextEpisode, //bound callback
+		closeOverlay //bound callback
```

Comment:
mmmm, I don't think these should be coming through as props? We can use `useContext` in the same way that we did in the other component - that way we can keep this component more modular and stuff. This is also quite a lot of props - are there any more than we can derive from the context instead of passing as a prop?

Reply:
I've redone the functions, the rest of the things being passed as props were themselves passed as props from Episodes -> Episode -> PreviewThumb, so we're just continuing the way we are really 🤷‍♂